Godeps/Godeps.json

```diff
@@ -198,6 +198,11 @@
 			"Rev": "535138d7bcd717d6531c701ef5933d98b1866257"
 		},
 		{
+			"ImportPath": "github.com/go-ini/ini",
```

Comment:
This block should be removed. It probably got pulled in by godeps automatically but it is already vendored correctly. We might move away from godeps to something that handles this case better. For now just manually remove it.